ironfish-cli/src/commands/wallet/send.ts

```diff
@@ -282,6 +287,11 @@ ${CurrencyUtils.renderIron(
       }
     }
 
+    if (flags.rawTransaction) {
+      this.log(`Raw transaction: ${rawTransactionResponse}`)
+      this.exit(0)
+    }
```

Comment:
Should this be before or after the confirm prompt?

I kind of think it makes more sense to put it before: the prompt says 'You are about to send . . .', but we're not about to send the transaction if we pass in `--rawTransaction`

Reply:
We still need to allow the user to confirm the transaction details even if its raw transaction. Otherwise the user would have no idea what is in the raw transaction. I will change the message from send to create transaction.

Reply:
updated